### PR TITLE
Minor grammatical fixes in subresources.md

### PR DESCRIPTION
**Pull request by warddave**
Minor grammatical fixes.

### CODE REVIEW

File: core/subresources.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Subresources work very well by implementing your own state [providers](./state-p
 or [processors](./state-processors.md). In API Platform we provide a working Doctrine layer for
 subresources providing you add the correct configuration for URI Variables.
 
-## URI Variables Configuration
+## URI Variable Configuration
```
Comment (alanpoulain) [inline core/subresources.md:11]: Why this change?

Comment (warddave) [reply core/subresources.md:11]: `Variable configuration` is a compound noun and in the case of a compound noun to make it plural, the last word uses the plural form and the first word uses the singular form..

e.g it's `Dog parks` not `dogs park`

So it'd be either `Variable configuration` (singular) or `Variable configurations` (plural).

But, in this case `variable configuration` is  slightly more natural because there is one type of configuration for the variables and the title refers to that one type of configuration, not the individual variables. 

`variable configurations` wouldn't be entirely incorrect either, but it's slightly awkward.

 `variables configuration` is definitely incorrect.

I'm a native english speaker 😄  - I had to think through the why for a second - it just sounded incorrect.
